js-intro.js

```diff
@@ -12,32 +12,61 @@ var temp2 = 350
 var temp3 = 212
 
 
+// create a function that takes in one argument
+// tells if the argument is above, below, or at 212
+const tempTest = (temp) => {
+    if (temp > 212) {
+        return `${temp} is above the boiling point.`
+    } else if (temp < 212) {
+        return `${temp} is below the boiling point.`
+    } else if (temp === 212) {
+        return "212 is at the boiling point."
+    }
+}
+console.log(tempTest(35));
+console.log(tempTest(350));
+console.log(tempTest(212));
 
 
 // --------------------2) Create a function that multiplies each number in the array by 5 using a for loop.
 // Use the test variable provided below. Expected outcome: [15, 35, 0, 30, -45]
 
 var myNumbers1 = [3, 7, 0, 6, -9]
 
-
-
+// create a function that takes in an arrray and multipies it by 5
+const multipier = (array) => {
+    let newArray = []
+    for (let i = 0; i < array.length; i++) {
+    newArray.push(array[i] * 5)
+    }
+    return newArray
+}
+console.log (multipier(myNumbers1))
 
 // --------------------3) Create a function that multiplies each number in the array by 5 using map.
 // Use the test variable provided below. Expected outcome: [40, -35, 0, 30, 10]
 
 var myNumbers2 = [8, -7, 0, 6, 2]
 
-
-
+const mapMultipier = (array) => {
+    let newArray = array.map (value => value * 5)
+    return newArray
+}
+console.log (mapMultipier(myNumbers2))
 
 // --------------------4) Create a function that removes all the vowels from a string.
 // Use the test variables provided below. Expected output: "HyThrLrnStdnt" "LvJvScrpt"
 
 var stringWithVowels1 = "HeyThereLearnStudent"
 var stringWithVowels2 = "ILoveJavaScript"
 
-
-
+const remover = (string) => {
+    let newArray1 = string.split("")
+    let newArray2 = newArray1.filter (value => value !== "a" && value !== "e" &&value !== "i" && value !== "o" && value !== "u" && value !== "I")
+    return newArray2.join("")
+}
```

Comment:
Consider how this solution might work if we passed in `var stringWithVowels3 = "EveryoneOut"